### ISSUE TITLE
Competitivo?

### ISSUE THREAD

**Issue by tobiasgz** [pdep-utn-frd/presentacion-2021-ValentinMartorelli#3]
Cuando decis que jugas titulos competitivos supongo que haces referencia a videojuegos, mi pregunta seria que titulos o genero?

**Comment by ValentinMartorelli** [comment]
Claro, juego Cs Go y LoL principalmente, tambien CoD Warzone, pero pase por varios battle royale tambien
